### PR TITLE
Update recipes to NXP kirkstone-5.15.52-2.1.0 

### PR DESCRIPTION
**Pull request by chrisdimich**

### CODE REVIEW

File: recipes-fsl/packagegroups/packagegroup-fsl-ml.bb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 DESCRIPTION = "Add packages for AI/ML build"
 
+PACKAGE_ARCH = "${TUNE_PKGARCH}"
```
Comment (otavio) [inline recipes-fsl/packagegroups/packagegroup-fsl-ml.bb:3]: Could it use MACHINE_SOCARCH?

File: recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch [hunk 2]
```diff
@@ -2,7 +2,7 @@ From 1ba241438dcc14852017757451521f5e27690290 Mon Sep 17 00:00:00 2001
 From: Cristinel Panfir <cristinel.panfir@nxp.com>
 Date: Mon, 2 May 2022 15:18:46 +0000
 Subject: [PATCH] Findtim-vx.cmake: Fix LIBDIR for multilib environment
-Upstream-Status: Pending
+
```
Comment (otavio) [inline recipes-libraries/tensorflow-lite/files/0001-Findtim-vx.cmake-Fix-LIBDIR-for-multilib-environment.patch:5]: Should still be sent upstream so header should be kept.

File: recipes-libraries/tensorflow-lite/tensorflow-lite-host-tools_2.9.1.bb [hunk 3]
```diff
@@ -26,5 +26,4 @@ do_configure:prepend() {
     export HTTPS_PROXY=${https_proxy}
     export http_proxy=${http_proxy}
     export https_proxy=${https_proxy}
-    
-}
+}
```
Comment (otavio) [inline recipes-libraries/tensorflow-lite/tensorflow-lite-host-tools_2.9.1.bb:29]: Missing new line at end of file

File: recipes-libraries/pytorch/pytorch_git.bb [hunk 4]
```diff
@@ -42,5 +42,3 @@ do_install(){
 }
 
 FILES:${PN} += "${libdir}/python*"
-
-COMPATIBLE_MACHINE = "(mx8-nxp-bsp)"
```
Comment (otavio) [inline recipes-libraries/pytorch/pytorch_git.bb:46]: Why?

File: recipes-libraries/pytorch/torchvision_git.bb [hunk 5]
```diff
@@ -30,5 +30,3 @@ do_install(){
 }
 
 FILES:${PN} += "${libdir}/python*"
-
-COMPATIBLE_MACHINE = "(mx8-nxp-bsp)"
```
Comment (otavio) [inline recipes-libraries/pytorch/torchvision_git.bb:34]: Why?

**Review by otavio — CHANGES_REQUESTED** [review]

**Comment by chrisdimich** [comment]
@otavio fixed all of these.

**Review by otavio — APPROVED** [review]

**Pull request merged**